Makefile

```diff
@@ -133,19 +133,35 @@ kustomize: ## Download kustomize locally if necessary.
 	$(call go-install,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.2)
 
 ##@ Package
-
-helm-package: helm-cmd helm-test
+helm-package-operator: helm-cmd helm-test
```

Comment:
should `helm-test` be renamed to `helm-operator-test` for consistency with the other targets?